pkgs/development/tools/cask/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, emacs, python }:
+
+stdenv.mkDerivation rec {
+  name = "cask-${version}";
+  version = "0.8.4";
+
+  src = fetchurl {
+    url = "https://github.com/cask/cask/archive/v${version}.tar.gz";
+    sha256 = "02f8bb20b33b23fb11e7d2a1d282519dfdb8b3090b9672448b8c2c2cacd3e478";
+  };
+
+  doCheck = true;
```

Comment:
btw, a possible build phase would be to compile the `*.el` files to `*.elc`, which would improve startup speed.